src/modules/postgres/setup/1.sql

```diff
@@ -44,37 +44,43 @@ CREATE TABLE items (
 	name text, -- room display name, thread title
 	body text, -- room description, thread start message
 	type smallint,
-	parents uuid[], -- room or thread
+	parents uuid[] NOT NULL, -- room or thread
 	tags smallint[], -- e.g. image, hidden, sticky, city, area, spot
-	createtime bigint,
+	createtime bigint DEFAULT extract(epoch from now()) NOT NULL,
 	creator text,
 	deletetime bigint,
 	meta jsonb, -- guides, image dimensions, counts
 	params jsonb,
 	terms tsvector,
 	updater text,
 	updatetime bigint,
-	counts jsonb default '{}'
+	counts jsonb DEFAULT '{}'
 );
 
 CREATE TABLE rooms (
+	name text NOT NULL,
```

Comment:
name is a property of the items table, and it is my understanding that it cannot be overwritten in an inherited table. have you tested this?